website/content/docs/runner/index.mdx

```diff
@@ -45,13 +45,21 @@ between operations, and allow Waypoint to scale further.
 
 On-demand runners: see [on-demand runners](/docs/runner/on-demand-runner) for more information.
 
+This diagram shows a CLI runner, as well as remote static and on-demand runners:
```

Comment:
Left a comment on Slack asking the best way to move forward (I can make a PR)

Some of the reorganizing of this sections in this file is undoing changes that are already live on https://developer.hashicorp.com/waypoint/docs/runner. Maybe things got out of sync since this was on `main` but not `stable-website`?

Reply:
merged in https://github.com/hashicorp/waypoint/pull/4163